icu4c/source/i18n/number_decimfmtprops.h

```diff
@@ -23,10 +23,14 @@ U_NAMESPACE_BEGIN
 #if U_PF_WINDOWS <= U_PLATFORM && U_PLATFORM <= U_PF_CYGWIN
 #if defined(_MSC_VER)
 // Ignore warning 4661 as LocalPointerBase does not use operator== or operator!=
-#pragma warning(suppress: 4661)
```

Comment:
It turns out that the `#pragma warning(suppress:)` only applies to the immediately next line... meaning the `#endif` in this case...
So we need to actually use the push/pop style here.